lib/linguist/languages.yml

```diff
@@ -7300,6 +7300,16 @@ UrWeb:
   tm_scope: source.ur
   ace_mode: text
   language_id: 383
+UxnTal:
+  type: programming
+  color: "#8dff9c"
```

Comment:
I think #72dec2 would look nicer :)

Reply:
Looks like that is free, I will go ahead and change it.

(@neauoire is the creator of Uxntal.)